use/posts.js

```diff
@@ -126,11 +129,9 @@ export const usePosts = () => {
       name: 'description',
       hideLabel: true,
       component: 'TInputTextarea',
-      placeholder: 'Text (markdown)',
-      tips:
-        'Pitch yourself: Who are you? What do you offer? What do you want?\n\nTips for effective pitch:\n- Uncomplicated: It should be catchy and roll off the tongue\n- Concise: It shouldn’t take more than a minute to say or read\n- Unique: It reflects your skills, goals, and desires\n- Storyline: It covers who you are, what you offer, and where you want to be\n- Appealing: Your elevator pitch is essentially a persuasive sales pitch; the emphasis should be on what you offer',
-      description:
-        'Use [widgets](https://wedance.vip/markdown), including images and videos',
+      placeholder: ('posts.edit.description.placeholderr'),
+      tips:t('posts.edit.description.tips'),
```

Comment:
Space missing. Do you have Prettier Extension installed in VSCode?

```suggestion
      tips: t('posts.edit.description.tips'),
```